common/djangoapps/student/signals/receivers.py

```diff
@@ -62,6 +62,9 @@ def create_course_enrollment_celebration(sender, instance, created, **kwargs):
     if not REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(instance.course_id):
         return
 
+    if not courseware_mfe_first_section_celebration_is_active(instance.course_id):
```

Comment:
Also, don't forget a test if one already exists for the `REDIRECT_TO_COURSEWARE_MICROFRONTEND` too.